portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
```

Comment:
it seems to me that you have two versions of the site here older: .../webapp/... and newer: .../webapp/html/... is it correct. If there is no reason for duplication please remove the redundant version.

Reply:
Remnants of refactoring html code into a specific folder. Whoops, when I committed I must've only added the new files, but forgot to delete the old ones.

Reply:
Done